js/form.js

```diff
@@ -0,0 +1,40 @@
+import {pristine} from './libs/pristin-init.js';
+
+import {resetButton, form} from './const.js';
+import {resetForm, blockSubmitButton, unblockSubmitButton} from './utils.js';
+import {sendData} from './api.js';
+
+// Отправляем данные из формы
+export const setUserFormSubmit = (onSuccess, onFail) => {
```

Comment:
```suggestion
export const setUserFormSubmit = (onSuccess, onError) => {
```

устоявшееся обозначение ошибки